src/lib/DataTable/DataTableInterfaces.ts

```diff
@@ -14,6 +14,7 @@ export interface DataTableRoutedProps<T, TFilter, TRouteName> extends CommonData
   data: TableQueryResult<T>;
   columns: DataTableColumnDescription<T>[];
   actions?: DataTableRoutedActions<T, TRouteName>;
+  actionsOnTheLeft?: boolean;
```

Comment:
I would prefer an Enum with `Left` and `Right` options. otherwise it's not clear to me as a user what actionsOnTheLeft=false does without reading the source code.

`actionsPosition: ActionsPosition`

Reply:
Converted from bool to enum :)